examples/demo.rs

```diff
@@ -1,3 +1,5 @@
+// Note: this requires the `derive` attribute
```

Comment:
Did you mean `feature` instead of `attribute`? Similarily below.

Reply:
Of course I had a typo like that.  Fixed